src/System.Xaml/System/Xaml/Replacements/TimeOnlyConverter2.cs

```diff
@@ -0,0 +1,100 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel;
+using System.ComponentModel.Design.Serialization;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+
+namespace System.Xaml.Replacements
+{
+    /// <summary>
+    /// Provides a type converter to convert <see cref='System.TimeOnly'/> objects to and from various other representations.
+    /// </summary>
+    public class TimeOnlyConverter2 : TypeConverter
```

Comment:
can this be internal? Also not convinced this is a good name.

Reply:
changed them to internal. The name is similar to the ones previously used eg. `DateTimeConverter2`